crates/consensus/primary/src/consensus/state.rs

```diff
@@ -234,31 +234,77 @@ impl ConsensusState {
 /// When a certificate gets committed then
 /// the corresponding certificate's round is considered a "committed" round. It bears both the
 /// committed round and the corresponding garbage collection round.
-#[derive(Debug, Default, Copy, Clone)]
+#[derive(Debug, Default, Clone)]
 pub struct ConsensusRound {
     pub committed_round: Round,
-    pub gc_round: Round,
+    pub gc_round: AtomicRound,
```

Comment:
I assume the need to make this atomic is not obvious from this PR?  Loosing Copy on a simple data structure like ConsensusRound seems suboptimal but I guess if there is some driver for it then it is what it is.
Also, it seems odd that one field is atomic and one is not.  Given the cloning the loss of copy forces not sure this is useful (or is that why AtomicRound has an Arc<Atomic>>?  Do you really just want to keep this copy and wrap in an Arc at a higher level?